test/general/key.js

```diff
@@ -3934,4 +4028,48 @@ XvmoLueOOShu01X/kaylMqaT8w==
     expect(signingKeySignature instanceof openpgp.SignaturePacket).to.be.true;
     expect(signingKeySignature.keyFlags[0] & openpgp.enums.keyFlags.signData).to.be.equals(openpgp.enums.keyFlags.signData);
   });
+
+  it("Sign and verify key - key with user attribute", async function () {
```

Comment:
Could you put the test case next to the other related ones, e.g. after the "Sign key and verify with wrong key - all users" one?

Reply:
ok, thank you.